src/Gui/Forms/MainFormInteractor.cs

```diff
@@ -529,6 +533,12 @@ public string ProjectFileName
             set { projectFileName = value; }
         }
 
+        public string ProjectOutputDir
+        {
+            get { return projectOutputDir; }
+            set { projectOutputDir = value; }
+        }
```

Comment:
`ProjectOutputDir` (and `projectOutputDir`) looks useless. I can't see any reading of this property (except line 702)

Reply:
`ProjectOutputDirectory` should be located in the `Project` class and should be used when generating the names of the output files that `Program` uses.